runner.py

```diff
@@ -51,9 +52,26 @@ def run_agent(terminate_event, callback_event, config_file, name, team, index, m
     bm = bot_manager.BotManager(terminate_event, callback_event, config_file, name, team,
                                 index, module_name)
     bm.run()
-
+    
+def inject()
```

Comment:
can we name this to
injectDLL

And add a comment in the format of 
"""
Comment here and says what it does and the situations under which it kills the app
"""

Reply:
ooo raising code standards on documentation :)